src/Symfony/Component/DependencyInjection/Argument/ServiceClosureArgument.php

```diff
@@ -14,7 +14,7 @@
 use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
 
 /**
- * Represents a service wrapped in a memoizing closure.
+ * Represents a service wrapped in a memorizing closure.
```

Comment:
This change is wrong (https://en.wikipedia.org/wiki/Memoization).

Reply:
Oh. Thank you. I learned something new today.